### ISSUE TITLE
Release Annotation not being automatically created for deployment

### ISSUE THREAD

**Issue by kyleburnsdev** [Azure/webapps-deploy#221]
When deploying application to App Service, release annotation is not being created in the associated Application Insights instance.

App Service Details:

- App Service Plan (B1) running Linux OS
- Attached Application Insights (Log Analytics Workspace)
- Java 11 runtime stack with integrated web server

Github workflow details:

- Github action using Azure credentials in stored secret (not Publish Profile)
- Jar file deployment

When deploying, a Release Annotation is not added to the Application Insights instance. Review of https://github.com/microsoft/pipelines-appservice-lib/blob/master/src/RestUtilities/AnnotationUtility.ts suggests that there should be console output at a minimum to show that annotation was attempted, but I see none of the logging messages in my workflow console output.

**Comment by shpraka** [comment]
@kyleburnsdev the log can be seen in debug mode. Here is an example of the run: https://github.com/shpraka/NodeJsAppVer18/actions/runs/4161350177/jobs/7199772309.  We can enable debug-logging by following instructions mentioned [here](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging).

**Comment by shpraka** [comment]
@kyleburnsdev Checking again.

**Comment by kyleburnsdev** [comment]
> @kyleburnsdev Checking again.

I'm sorry for the delay, @shpraka, but didn't realize you were waiting on something from me. Are you  wanting me to run with debug enabled and provide the output or are you suggesting this has been repaired and asking for confirmation on my end?

**Comment by shpraka** [comment]
@kyleburnsdev I was waiting for the confirmation if you are able to see the release annotations when you enable debug logging. Once you confirm, we can close the issue.

**Comment by kyleburnsdev** [comment]
> @kyleburnsdev I was waiting for the confirmation if you are able to see the release annotations when you enable debug logging. Once you confirm, we can close the issue.

Thanks for the clarification. I will set up a test and let you know the results

**Comment by shpraka** [comment]
Closing this issue because of no issue in last 7 days. Feel free to re-open or create a new issue.
